perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/ServicesManagerMethod.java

```diff
@@ -371,7 +371,8 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 			parms.stateChangingCheck();
 
 			ac.getServicesManager().deleteService(ac.getSession(),
-					ac.getServiceById(parms.readInt("service")));
+					ac.getServiceById(parms.readInt("service")),
+					parms.readBoolean("force"));
```

Comment:
This actually makes the `force` parameter required. The method `readBoolean` throws an exception if the parameter is not found. To check if a parameter is present you can call the `parms.contains()` method.

Reply:
Good point. Fixed.